test/utilities.js

```diff
@@ -230,6 +230,8 @@ describe('utilities', function () {
     });
 
     expect(expect('foo').result()).to.equal('result');
+
+    delete chai.Assertion.prototype.result;
```

Comment:
Of course, you're absolutely right. Thanks for noticing.
I'll make the changes.